src/redux/reducers/cellInfo/updateCellInfo.js

```diff
@@ -0,0 +1,11 @@
+/* eslint-disable no-param-reassign */
+import produce from 'immer';
+
+import initialState from './initialState';
+
+const updateCellInfo = produce((draft, action) => {
+  const { cellId } = action.payload;
+  draft.cellId = cellId;
+}, initialState);
+
+export default updateCellInfo;
```

Comment:
it makes sense to still call it `cellInfoUpdate`, so that it is consistent with the rest of the cellInfo actions